### PR TITLE
Release notes v0.13.1.0

### PR DESCRIPTION
**Pull request by UdjinM6**
```
 rewrite doc/release-notes.md (94%)
 copy doc/{release-notes.md => release-notes/dash/release-notes-0.13.0.md} (100%)
```

### CODE REVIEW

**Comment by codablock** [comment]
Should probably note that there is no proto bump involved and no `masternode start` being required.

File: doc/release-notes.md [hunk 1]
```diff
@@ -37,653 +41,50 @@ through a BIP9 (bit 3) deployment.
...
+
+Bug fixes/Other improvements
+----------------------------
+There are few bug fixes in this release:
+- Block size should be calculated correctly for blocks with quorum commitments when constructing new block;
+- Special transactions should be checked for mempool acceptance at the right time (nodes could ban each other
+in some rare cases otherwise);
+- Signature verification and processing of special transactions is decoupled to avoid any potential issues.
 
 0.13.0.0 Change log
```
Comment (codablock) [inline doc/release-notes.md]:
```suggestion
 0.13.1.0 Change log
```

Comment (codablock) [reply doc/release-notes.md]: Hmm...not sure the suggestion will do what I intended...When I initially created it, it tried to remove a non-existent space at the start of the line...

**Comment by codablock** [comment]
Haha, you've update the PR faster then I commented about missing commits ;)
So it's only the inline-comment/suggestion that's left

**Comment by UdjinM6** [comment]
Should be up to date and include all fixes now :)

**Pull request merged**